### PR TITLE
Document manual node setup with existing KMS keys

### PR DESCRIPTION
**Pull request by JoaoSa-da**

### CODE REVIEW

File: docs/2.7.0/docs/canton/usermanual/security.rst [hunk 1]
```diff
@@ -550,6 +550,24 @@ AWS KMS only supports the :ref:`following cryptographic schemes <canton_supporte
...
+
+We then need to register the keys in Canton by running the key registration command on each node.
+For example for a participant we would run:
+
+.. literalinclude:: /canton/includes/mirrored/enterprise/app/src/test/resources/kms-provider-tagged.conf
+   :language: scala
+   :start-after: user-manual-entry-begin: ManualRegisterKmsKeys
+   :end-before: user-manual-entry-end: ManualRegisterKmsKeys
+
+Finally, we need to initialize our :ref:`domain <_manually-init-domain>` and
```
Comment (soren-da) [inline docs/2.7.0/docs/canton/usermanual/security.rst:568]: those steps still include the generation of keys. can you refactor them such that either the three keys are generated or registered at the beginning, and then all follow-up steps are independent of the key setup step. and include the common steps also here

Comment (soren-da) [reply docs/2.7.0/docs/canton/usermanual/security.rst:568]: ah right, my bad, I was looking at the v2.6.5 docs

File: docs/2.7.0/docs/canton/usermanual/security.rst [hunk 2]
```diff
@@ -550,6 +550,24 @@ AWS KMS only supports the :ref:`following cryptographic schemes <canton_supporte
...
 
+In the previous example, Canton creates its own keys on startup. If we want to create the keys ourselves and pass then
+to Canton we have to manually start the nodes by adding the following flag in the config:
+
+.. code-block:: none
+
+    <node>.init.auto-init = false
+
+We then need to register the keys in Canton by running the key registration command on each node.
+For example for a participant we would run:
```
Comment (soren-da) [inline docs/2.7.0/docs/canton/usermanual/security.rst:561]: please document what `namespaceKmsKeyId` and friends needs to be, e.g., a AWS KMS Key ARN

File: docs/2.7.0/docs/canton/usermanual/security.rst [hunk 3]
```diff
@@ -550,6 +550,24 @@ AWS KMS only supports the :ref:`following cryptographic schemes <canton_supporte
     provider it can no longer be reverted without a full reset of the node
     (i.e., re-generation of node identity and all keys).
 
+In the previous example, Canton creates its own keys on startup. If we want to create the keys ourselves and pass then
```
Comment (soren-da) [inline docs/2.7.0/docs/canton/usermanual/security.rst:553]:
```suggestion
In the previous example, Canton creates its own keys on startup and initialized the identity of the nodes automatically. If the keys have already been generated in the KMS, we need to manually initialize the identity of the nodes by adding the following flag in the config:
```

File: docs/2.7.0/docs/canton/usermanual/security.rst [hunk 4]
```diff
@@ -550,6 +550,24 @@ AWS KMS only supports the :ref:`following cryptographic schemes <canton_supporte
...
+to Canton we have to manually start the nodes by adding the following flag in the config:
```
Comment (soren-da) [inline docs/2.7.0/docs/canton/usermanual/security.rst:554]:
```suggestion
```

**Review by soren-da — APPROVED** [review]

**Review by carrielaben-da — APPROVED** [review]
Looks good!

**Pull request merged**